common/src/com/intellij/plugins/haxe/util/HaxeCommandLine.java

```diff
@@ -30,25 +30,35 @@
 
   public HaxeCommandLine(@NotNull Module module) {
     super();
+    addEnvironmentVars();
     patchPath(module);
   }
 
   public HaxeCommandLine(@NotNull Module module, @NotNull String... command) {
     super(command);
+    addEnvironmentVars();
```

Comment:
Isn't this call unnecessary because it's already called from super constructor?

Reply:
As far as i know `super(command)` would be calling the `GeneralCommandLine` constructor, not any of the other `HaxeCommandLine` constructors, for that to be the case it would have to be  `this(command)`

Reply:
Ah, gosh, sure it is :-)